domains/measurements/Measurements.API/API/Measurements/Helpers/TimeSeriesHelper.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Globalization;
+using Measurements.V1;
+
+namespace API.Measurements.Helpers
+{
+    public static class TimeSeriesHelper
```

Comment:
i am guessing all of this is some sort of datahub and how they handle time. Do you know what it all means?

Reply:
It is a helper for MeterTimeSeries yes (which is the response from DataHubFacade). It is taken from DataSync.API. I know what they mean, yes